tests/smoke/test_homepage.py

```diff
@@ -0,0 +1,33 @@
+"""Smoke tests for addons-frontend docker image."""
+
+import pytest
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+
+
+@pytest.fixture
+def firefox_options(firefox_options):
+    """Set up Firefox options."""
+    firefox_options.add_argument("-headless")
+    firefox_options.add_argument('-foreground')
+    return firefox_options
+
+
+@pytest.fixture
+def selenium(base_url, selenium):
+    """Set up selenium."""
+    # Simple wait for homepage to load
+    selenium.get(base_url)
+    WebDriverWait(selenium, 60).until(
+        EC.visibility_of_element_located(
+            (By.CSS_SELECTOR, ".Hero-contents")
```

Comment:
Is it too hard to look for the comment `<!-- Godzilla of browsers -->` instead? The ops teams already uses that comment to detect successful deployments (iirc); it won't change. The `.Hero-contents` selector might change.

Reply:
No its not. I totally forgot to change this

Reply:
Also I don't think selenium can find comments.

Reply:
Is finding that element not good enough?